components/suggest/src/provider.rs

```diff
@@ -45,6 +47,22 @@ impl SuggestionProvider {
             _ => None,
         }
     }
+
+    pub(crate) fn records_for_provider(&self) -> Vec<SuggestRecordType> {
+        match self {
+            SuggestionProvider::Amp => {
+                vec![SuggestRecordType::AmpWikipedia, SuggestRecordType::Icon]
+            }
+            SuggestionProvider::Wikipedia => {
+                vec![SuggestRecordType::AmpWikipedia, SuggestRecordType::Icon]
+            }
+            SuggestionProvider::Amo => vec![SuggestRecordType::Amo],
+            SuggestionProvider::Pocket => vec![SuggestRecordType::Pocket],
+            SuggestionProvider::Yelp => vec![SuggestRecordType::Yelp],
+            SuggestionProvider::Mdn => vec![SuggestRecordType::Mdn],
+            SuggestionProvider::Weather => vec![SuggestRecordType::Weather],
+        }
```

Comment:
I see that we have a new `SuggestRecord` type called `GlobalConfig`. Which providers should it be mapped with?